orchestra/tests/helpers/__init__.py

```diff
@@ -73,3 +74,17 @@ def assertModelInstanceExists(self,
     def assertModelInstanceNotExists(self, model_class, lookup_attributes):
         with self.assertRaises(model_class.DoesNotExist):
             model_class.objects.get(**lookup_attributes)
+
+
+class OrchestraTestCase(OrchestraTestHelpersMixin, TestCase):
+    # NOTE(lydia): Mixin should go before TestCase because when mixin calls
```

Comment:
@marcua @joshblum good thing we spent a while figuring out multiple inheritance in python! also http://nedbatchelder.com/blog/201210/multiple_inheritance_is_hard.html